### PR TITLE
Chat window implemented

### PR DESCRIPTION
**Pull request by linhapeter**

### CODE REVIEW

File: src/messenger/Chat.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      console.log(messages);
+      e.target.value = "";
+    }
+  }
+
+  return (
+    <div className="chat">
+      <div className="message-list">
+        {messages.map((message, index) => (
+          <Message key={index} text={message.text} />
```
Comment (volkov-a-dev) [inline src/messenger/Chat.jsx]: Using an index on a given key is bad practice. Please read the [docs](https://react.dev/learn/rendering-lists#rules-of-keys)

File: src/messenger/Chat.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  const changeInputMessage = (e) => {
+    setMessageInput(e.target.value);
+    console.log(messageInput);
+  }
+
+  const sendMessageOnEnter = (e) => {
+    if (e.key === 'Enter') {
+      setMessages([...messages, { text: messageInput }]);
+      setMessageInput("");
+      console.log(messages);
```
Comment (volkov-a-dev) [inline src/messenger/Chat.jsx]: Please remove/comment out the console log

File: src/messenger/Chat.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import Message from './Message';
+
+const Chat = () => {
+  const [messageInput, setMessageInput] = useState("");
+  const [messages, setMessages] = useState([]);
+
+  const changeInputMessage = (e) => {
+    setMessageInput(e.target.value);
+    console.log(messageInput);
```
Comment (volkov-a-dev) [inline src/messenger/Chat.jsx]: Please remove/comment out the console log

File: src/messenger/Chat.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  return (
+    <div className="chat">
+      <div className="message-list">
+        {messages.map((message, index) => (
+          <Message key={index} text={message.text} />
+        ))}
+      </div>
+      <input className="input-message" type="text" onChange={changeInputMessage} onKeyPress={sendMessageOnEnter} />
```
Comment (volkov-a-dev) [inline src/messenger/Chat.jsx]: Why is CSS used? Why is the file here?

File: src/messenger/Chat.jsx
Comment (volkov-a-dev) [inline src/messenger/Chat.jsx:1]: Why is business logic not separated from rendering?

Comment (volkov-a-dev) [reply src/messenger/Chat.jsx:1]: Do you have an eslint error in this file?

File: src/App.jsx [hunk 5]
```diff
@@ -1,9 +1,13 @@
 import React from "react";
-import Form from "./Form.jsx";
+import Form from "./form/Form";
+import Chat from "./messenger/Chat";
+import "./style.css";
```
Comment (volkov-a-dev) [inline src/App.jsx:4]: Why is there CSS here?

**Review by volkov-a-dev — CHANGES_REQUESTED** [review]

File: src/App.jsx (hunk 5 above)
Comment (linhapeter) [reply src/App.jsx:4]: I'don't understand where should be css?

Comment (volkov-a-dev) [reply src/App.jsx:4]: I expressed myself incorrectly. This style.css file must have global styles.
Styles intended for a component must be at the component level.

**Pull request merged**